CourseApp/Tasks.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace CourseApp
+{
+    using System;
+
+    public class Tasks
+    {
+        public double Function(double x = 0)
+        {
+            double sinAngle = Math.Sin(x);
+            double cosAngle = Math.Cos(x);
+            return ((sinAngle * sinAngle * sinAngle) + (cosAngle * cosAngle * cosAngle)) * Math.Log(x);
+        }
```

Comment:
Duplicated, this code already exists in `Equation` class

Reply:
@DustofTimeYT - Please fix review points